### ISSUE TITLE
DNS Root Migration

### ISSUE THREAD

**Issue by kj4ezj** [eosnetworkfoundation/engineering#11]
Prepare for and participate in the DNS root migration of some of our key domains.

**Comment by kj4ezj** [comment]
Today's meeting was postponed. We may do this asynchronously instead of on a video call, not sure.

**Comment by kj4ezj** [comment]
@tedcahalleos and I migrated the `SOA`, `NS`, and `A` records for `eosnetwork.com.` from Google Domains to AWS Route 53 this morning around 2022-12-17 11:12 EST. We were able to verify that `dig` is returning the expected records and tested the website at [eosnetwork.com](https://eosnetwork.com), [docs.eosnetwork.com](https://docs.eosnetwork.com), and [eosnetwork.com](https://learn.eosnetwork.com) from Blacksburg VA, Manassas VA, Israel, and Hong Kong. We also tested `api.learn.eosnetwork.com` with `curl`, and were able to send email to our domain using at least three different external providers.

I put this ticket in `Awaiting Review` to indicate the work is done. I will close on Monday if there are no issues reported.
